tools/cli/commands-cordova.js

```diff
@@ -131,6 +133,7 @@ main.registerCommand({
   requiresApp: true,
   catalogRefresh: new catalog.Refresh.Never()
 }, function (options) {
+  ensureDevBundleDependencies();
```

Comment:
Yeah, I should have notated this commit to explain its seemingly silly, preparatory nature.  The diff just looked rather obnoxious without breaking it down this way.